autoload/vebugger.vim

```diff
@@ -268,26 +270,39 @@ function! vebugger#createDebugger(command)
     return l:debugger
 endfunction
 
+if has('timers')
+    function! ReadingTimerCallback(debugger, timer_id)
```

Comment:
The timer callback part seems to be a bit tricky. I had some problems to get it running at all, since `start_timer` does not accept script local functions directly (it gives `E120: Using <SID> not in a script context`). But you are right, it is not necessary to have both a local function and a partial. After some tests I found out: 

You can use either use

- a global function and pass it to `start_timer` or
- a script local function and pass a partial to `start_timer`. You are right, there is no need to bind the `s:debugger` variable.

First I thought that timer callbacks mustn't access any script local attributes; so I coded that ugly solution with a global function and a partial.

Isn't it possible at all to use partials (or anything similiar) in neovim? It would be quite ugly to pick the solution with a global function.